codes/go/chapter_computational_complexity/leetcode_two_sum.go

```diff
@@ -0,0 +1,26 @@
+package chapter_computational_complexity
+
+// twoSumBruteForce
+func twoSumBruteForce(nums []int, target int) []int {
+	size := len(nums)
+	for i := 0; i < size-1; i++ {
+		for j := i + 1; i < size; j++ {
+			if nums[i]+nums[j] == target {
+				return []int{i, j}
+			}
+		}
+	}
+	return nil
+}
+
+// twoSumHashTable
+func twoSumHashTable(nums []int, target int) []int {
+	hashTable := map[int]int{}
+	for idx, val := range nums {
+		if preIdx, ok := hashTable[target-val]; ok {
+			return []int{preIdx, idx}
```

Comment:
We prefer `space` to `tab` (equivalent to 4 spaces) to avoid strange bugs.

Reply:
same as above